DNN Platform/DotNetNuke.Web/InternalServices/ControlBarController.cs

```diff
@@ -637,6 +651,13 @@ private int DoAddExistingModule(int moduleId, int tabId, string paneName, int po
                 }
                 else
                 {
+                    //copy tab module settings
```

Comment:
I believe the settings should be copied only when copy checkbox is checked. So there is not need to copy settings in else part.